source/common/quic/envoy_quic_dispatcher.cc

```diff
@@ -107,5 +108,26 @@ quic::QuicConnectionId EnvoyQuicDispatcher::ReplaceLongServerConnectionId(
   return new_connection_id;
 }
 
+void EnvoyQuicDispatcher::closeConnectionsWithFilterChain(
+    const Network::FilterChain* filter_chain) {
+  if (connections_by_filter_chain_.find(filter_chain) != connections_by_filter_chain_.end()) {
```

Comment:
This code looks up `filter_chain` twice. Keep the result of `find` so it only needs to be looked up once.

Reply:
done